src/components/course/ExpiredSubscriptionModal.jsx

```diff
@@ -0,0 +1,28 @@
+import {
+  StandardModal, useToggle,
+} from '@openedx/paragon';
+import { Link } from 'react-router-dom';
+import { useSubscriptions } from '../app/data';
+
+const ExpiredSubscriptionModal = () => {
+  const { data: { customerAgreement } } = useSubscriptions();
+  const [isOpen, ,close] = useToggle(true);
```

Comment:
Curious, why we have double `,` here?

Reply:
useToggle returns three things, isOpen, open, close. As I only need isOpen and close, So I skipped the open function and destructured my required ones. It was necessary to follow this pattern otherwise modal wasn't working as expected.

Reply:
@jajjibhai008 I think it's better to use `_` here to clearly indicate that the second value is intentionally ignored, rather than accidentally skipped. The `_` acts as a placeholder for the second element, open, which you don't need.

`const [isOpen, _, close] = useToggle(true);`

Reply:
Actually the format I used is already being used in the code, so I just followed that. If you think it's necessary to change that then I could.